src/lib/components/Canvas.svelte

```diff
@@ -46,6 +48,14 @@
 			return true;
 		});
 	}
+
+	cursorPosition.subscribe(() => {
+		let { x, y } = viewportToCanvasCoordinatesFromCanvasView($cursorPosition, $canvasView);
+		$user.cursorPosition = {
+			x: x,
+			y: y
+		};
```

Comment:
Hvis du ændrer typen på cursorPosition som jeg beskrev længere nede, så kan du slippe for det her og bare sætte den lig med da de vil have samme type